operators/pkg/instctrl/controller_test.go

```diff
@@ -86,11 +86,24 @@ var _ = Describe("The instance-controller Reconcile method", func() {
 			ObjectMeta: metav1.ObjectMeta{Name: webdavSecretName, Namespace: testName},
 			Data:       map[string][]byte{"username": []byte(testName), "password": []byte(testName)},
 		}
+		tenantPvcSecret := corev1.Secret{
+			ObjectMeta: metav1.ObjectMeta{Name: tenantPvcSecretName, Namespace: testName},
+			Data: map[string][]byte{
+				tenantPvcSecretServerIpKey:   []byte(testName),
+				tenantPvcSecretServerNameKey: []byte(testName),
+				tenantPvcSecretPathKey:       []byte(testName),
+			},
+		}
 		tenant := clv1alpha2.Tenant{
 			ObjectMeta: metav1.ObjectMeta{Name: testName},
 			Spec: clv1alpha2.TenantSpec{
 				Email: "test@email.me",
 			},
+			Status: clv1alpha2.TenantStatus{
+				PersonalNamespace: clv1alpha2.NameCreated{
+					Name: testName,
+				},
+			},
```

Comment:
This should not be required anymore (since the instance should not check this), not sure though.

Reply:
Ok, tried to remove it. If it will be required I will add it again